maps/away/blueriver/blueriver.dm

```diff
@@ -114,94 +117,202 @@
+			SPAN_WARNING("You draw away from \the [src].")
+		)
+		return
+	chomp_chomp_yummy_explorers(user)
 
-/obj/structure/deity
-	icon = 'icons/obj/cult.dmi'
-	icon_state = "tomealtar"
-	var/health = 10
-	density = TRUE
-	anchored = TRUE
+/turf/unsimulated/floor/bluespace_river/attack_robot(mob/user)
+	attack_hand(user)
 
-/obj/structure/deity/attackby(obj/item/W as obj, mob/user as mob)
-	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
-	user.do_attack_animation(src)
-	playsound(get_turf(src), 'sound/effects/Glasshit.ogg', 50, 1)
-	user.visible_message(
-		"<span class='danger'>[user] hits \the [src] with \the [W]!</span>",
-		"<span class='danger'>You hit \the [src] with \the [W]!</span>",
-		"<span class='danger'>You hear something breaking!</span>"
+/turf/unsimulated/floor/bluespace_river/attack_generic(mob/user)
+	attack_hand(user)
+
+/turf/unsimulated/floor/bluespace_river/Crossed(atom/movable/M)
+	. = ..()
+	if (locate(/obj/structure/catwalk) in src || !can_erase(M))
+		return
+	chomp_chomp_yummy_explorers(M)
+
+/// Returns whether or not this blue river turf can consume an item.
+/turf/unsimulated/floor/bluespace_river/proc/can_erase(atom/A)
+	if (ismovable(A))
+		var/atom/movable/AM = A
+		if (AM.throwing) // wait 'til objects and mobs finish flying through the air
+			if (AM.throwing.target_turf == src)
+				return TRUE
+			else
+				return FALSE
+	return !QDELETED(A) && A.simulated && !isobserver(A) && !istype(A, /obj/item/projectile) && has_gravity(A)
+
+/// Deletes the provided atom from existence forever with a spooky message. Mobs will leave behind an afterimage that slowly fades away.
+/turf/unsimulated/floor/bluespace_river/proc/chomp_chomp_yummy_explorers(atom/movable/M)
+	if (!can_erase(M))
+		return
+	if (isliving(M))
+		var/mob/living/L = M
+		L.visible_message(
+			SPAN_DANGER("\The [L] makes contact with \the [src]... and then goes completely still."),
+			SPAN_DANGER(FONT_LARGE("You [L.loc == src ? "fall into" : "touch"] \the [src]. Instantly, you feel a tingling, starting inside of you, and then spreading out. \
+			As your mind and body are completely erased from existence, you realize: you never were the real one, were you?")),
 		)
-	take_damage(W.force)
 
-/obj/structure/deity/take_damage(var/amount)
-	health -= amount
-	if(health < 0)
-		src.visible_message("\The [src] crumbles!")
-		qdel(src)
+		// Create an afterimage of the mob that slowly fades off
+		// The actual mob is instantly deleted - this is just for effect
+		// It's inspired by what an observer theoretically sees when something passes the event horizon of a black hole!
+		var/obj/effect/temporary/T = new (get_turf(L), 30 SECONDS)
+
+		T.icon = image(M)
+		T.icon_state = M.icon_state
+		T.overlays = M.overlays.Copy()
+		T.layer = MOB_LAYER
+		T.set_dir(M.dir)
+
+		T.anchored = TRUE
+		T.name = M.name
+		T.gender = M.gender
+		T.desc = "A picture-perfect image of [T], \his body and expression unmoving. They are slowly fading away, into nothing."
```

Comment:
"...Urist McGreytide, his body and expression unmoving. They are slowly..." sounds odd because of the sudden change in gender. Here are two suggestions either way, you need the `\the`).

```suggestion
		T.desc = "A picture-perfect image of \the [T], their body and expression unmoving. They are slowly fading away into nothing."
```

```suggestion
		T.desc = "A picture-perfect image of \the [T], \his body and expression unmoving. \He [T.gender == PLURAL ? "are" : "is"] slowly fading away into nothing."
```

Reply:
I'll use the gender datums here instead, as that way it'll be cleaner/not specifically have a check for plural.